[DOC] Restructure user documentation (#945)
saros-project/saros: docs/documentation/how-tos/awareness-information.md

---
title: Awareness Information
---

> ## Follow-Mode

Reviewer: :+1: Seems like a good overview of the available awareness information. How about adding a link to this page to the "Awareness Information" section on the landing page?

It should be mentioned that the "Follow-Mode", "File Navigation", and "Viewports" sections currently only apply to Saros/E as these features have not yet been implemented for Saros/I.

## File Navigations

In the package explorer:
- ![active file](images/icons-e/active_file.png) A colored dot decorates the file that a participant has currently
    in focus. The color resembles the color of the active user. If multiple users have the same file focused,
    the dot will turn grey (![multiple active file](images/awareness/eclipse_package_manager_annotated_file_multiple.png)).
- ![shared file](images/icons-e/shared_file.png) A
    blue arrow decorates a file that is shared with
    other participants.

![editor focus](images/awareness/eclipse_session_list_focused_editor.png)

In the Saros view's session list shows which participant has which editor focused.

## Cursors

![cursor annotation](images/awareness/eclipse_editor_cursor_annotation.png)

The position of a participant's cursor appears in the file and
the character behind the cursor is highlighted with the
corrsponding color.

## Selections

![selection annotation](images/awareness/eclipse_editor_selection_annotation.png)

Any text selected by a participant also appears highlighted in
the file.

## Changes

![contribution annotation](images/awareness/eclipse_editor_contribution_annotation.png)

> If a participant writes something, his/her text will appear
> highlighted in his/her color.

Reviewer: Again, personal preference.

```suggestion
If a participant writes something, their text will appear
highlighted in their color.
```
